src/server.c

```diff
@@ -1126,6 +1110,40 @@ void clientsCron(void) {
     }
 }
 
+/* A periodic timer that performs client maintenance.
+ * This cron task follows the following rules:
+ *  - All clients need to be checked (at least) once per second
+ *  - To manage latency, we don't check more than MAX_CLIENTS_PER_CLOCK_TICK at a time
+ *  - The minimum rate will be defined by server.hz
+ *  - At least CLIENTS_CRON_MIN_ITERATIONS will be performed each cycle
+ */
+#define CLIENTS_CRON_MIN_ITERATIONS 5
+long long clientsTimerProc(struct aeEventLoop *eventLoop, long long id, void *clientData) {
```

Comment:
we should also count the time into `server.el_cron_duration`

Reply:
1. in `serverCron` we get `server.el_cron_duration += serverCron`
2. in `clientsCron` we get `server.el_cron_duration += clientsCron`
3. in `beforeSleep` we get `server.el_cron_duration += duration_before_aof + duration_after_write` and use `durationAddSample` to accumulate the total duration, and reset `server.el_cron_duration` to `0`

Reply:
done.